src/com/facebook/buck/jvm/java/intellij/AbstractIjProjectConfig.java

```diff
@@ -48,6 +48,10 @@ public boolean isAutogenerateAndroidFacetSourcesEnabled() {
 
   public abstract Optional<String> getProjectJdkType();
 
+  public abstract Optional<String> getProjectAndroidSdkName();
```

Comment:
`Project` means it's a project level setting. In this case methods should be `getAndroidLibrarySdkName` and `getAndroidLibrarySdkType`.